Configure

```diff
@@ -2673,6 +2673,12 @@ make)
 	gmake)
 	echo "I can't find make or gmake, and my life depends on it." >&4
 	echo "Go find a public domain implementation or fix your PATH setting!" >&4
+
+	test -f /etc/debian_version && echo "Run \"apt install build-essential\" to install make and friends" >&4
```

Comment:
I can change the command to `apt-get` to include the legacy systems as well, but I don't think it's necessary since `apt` is the way to go, and users are expected to update their package manager anyway.

- https://itsfoss.com/apt-vs-apt-get-difference/

Targetting old stuff to the harm of the new is a bad thing to do IMHO.